nevermined_sdk_py/nevermined/assets.py

```diff
@@ -647,6 +657,23 @@ def _build_access(metadata, publisher_account, asset_rewards):
             "_receivers": asset_rewards["_receivers"]
         }}
 
+    @staticmethod
+    def _build_access_proof(metadata, publisher_account, asset_rewards):
+        return {"main": {
+            "name": "dataAssetAccessProofServiceAgreement",
+            "creator": publisher_account.address,
+            "price": metadata[MetadataMain.KEY]['price'],
+            "timeout": 3600,
```

Comment:
This should  be configurable, perhaps this is also an issue in some other similar methods. If not only access proofs can be fulfilled in one hour